### DISCUSSION TITLE
Generate executable for ubuntu using Avalonia framework in windows visual studio 2019

### DISCUSSION THREAD

**Discussion by santhoshkumarmk** [AvaloniaUI/Avalonia#6415] [Q&A]
Hi All,

I am developing Avalonia Application using Visual studio 2019 on Windows machine .  Now my question is how to generate bin files like EXE used in windows machine .  how to generate files for ubuntu machine to run application.

**Comment by EtherGhost** [comment]
I tried that the other day and it was easy and worked fine.
I just right clicked on the project in VS and then publish. After that publish to folder.
My profile settings where: Release x64, net5.0, framework-dependent, linux-x64, produce single file, but you should of cause use profile settings that suits your case. 
After that it was just a matter of copying the files created to your linux machine and making the file executable.
